defaults/main.yml

```diff
@@ -30,15 +30,15 @@ haproxy_logrotate_retention: 5
 ##### HAproxy configuration file
 # Global
 haproxy_global_log_socket: 127.0.0.1
-haproxy_global_log_facility: local0
+haproxy_global_log_facility: local5
 haproxy_global_log_level: debug
 
 haproxy_global_chroot: /var/lib/haproxy
 
 haproxy_global_user: haproxy
 haproxy_global_group: haproxy
 
-haproxy_global_stats_socket: /run/haproxy/haproxy.sock mode 0660 level admin
+haproxy_global_stats_socket: /run/haproxy/stats.sock mode 0660 level admin
```

Comment:
Is the "mode 0660 level admin" thing used somewhere ? Looks like you always ".split(' ')[0]" it away...
Where is the format of this string described ?

Reply:
I just do not want to break a backward compatibility